app/helpers/billing_helper.rb

```diff
@@ -0,0 +1,21 @@
+module BillingHelper
+  private
+
+  def default_plan?(account)
+    installation_config = InstallationConfig.find_by(name: 'CHATWOOT_CLOUD_PLANS')
+    default_plan = installation_config&.value&.first
```

Comment:
@scmmishra I added this case as this installation config is not available in the self-hosted installations.